knative-operator/deploy/resources/kn-downloads.yaml

```diff
@@ -0,0 +1,155 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+ name: serverless-operator
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: serverless-operator
+  name: kn-downloads
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: serverless-operator
+      component: kn-downloads
+  template:
+    metadata:
+      name: kn-downloads
+      labels:
+        app: serverless-operator
+        component: kn-downloads
+    spec:
+      nodeSelector:
+        kubernetes.io/os: "linux"
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: "NoSchedule"
+      - key: "node.kubernetes.io/unreachable"
+        operator: "Exists"
+        effect: "NoExecute"
+        tolerationSeconds: 120
+      - key: "node.kubernetes.io/not-ready"
+        operator: "Exists"
+        effect: "NoExecute"
+        tolerationSeconds: 120
+      containers:
+      - name: kn-download-server
+        terminationMessagePolicy: FallbackToLogsOnError
+        image: quay.io/openshift-knative/kn-cli-artifacts:latest
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 8080
+          name: http
+          protocol: TCP
+        livenessProbe:
+          httpGet:
+            path: /
+            port: 8080
+            scheme: HTTP
+        readinessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /
+            port: 8080
+            scheme: HTTP
+        command: ["/bin/sh"]
+        resources:
+          requests:
+            cpu: 10m
+            memory: 50Mi
+        args:
+        - '-c'
```

Comment:
Wouldn't this be easier with an nginx image or something like that?

Reply:
Yes, however I re-used most of these specs from `oc` `ConsoleCLIDownload which is being shipped/tested. We can refine this in next iterations once we've this functioning end to end.